ports/mimxrt/boards/MIMXRT1064_EVK/MIMXRT1064_EVK_af.csv

```diff
@@ -0,0 +1,21 @@
+Pad,ALT0, ALT1, ALT2, ALT3, ALT4, ALT5, ALT6, ALT7, ALT8, ALT9,ADC,ACMP,Default
+GPIO_AD_B1_07,FLEXSPI_B_DATA0,LPI2C3_SCL,LPUART3_RXD,SPDIF_EXT_CLK,CSI_HSYNC,GPIO1_IO23,USDHC2_DATA3,KPP_COL4,GPT2_COMPARE3,FLEXIO3_D07,ADC1_IN12,ACMP1_IN5,ALT5
```

Comment:
as above, this can be moved up a directory

Reply:
see above